spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/services/StatusUpdater.java

```diff
@@ -59,16 +60,22 @@ public class StatusUpdater {
 
 	private final ApiMediaTypeHandler apiMediaTypeHandler;
 
+	private Duration timeout = Duration.ofSeconds(10);
```

Comment:
Default timeout is 10 seconds as the default interval check in `IntervalCheck` is 10 seconds as well. Can be overridden by `timeout` method.